composer.json

```diff
@@ -11,6 +11,7 @@
         "ezsystems/platform-ui-assets-bundle": "^2.0.0",
         "ezsystems/repository-forms": "^1.0",
         "ezsystems/ezpublish-kernel": "^6.2",
+        "ezsystems/ez-support-tools": "^0.1-dev",
```

Comment:
If ez-support-tool is not meant to make it to 1.0.0-beta until next monday, then I suggest we hold this PR back until after 16.04 freeze.